WalletWasabi/WabiSabi/Models/DecompositionAlgs/Coin.cs

```diff
@@ -0,0 +1,40 @@
+using NBitcoin;
+using System;
+
+namespace WalletWasabi.WabiSabi.Models.DecompositionAlgs
+{
+	public record Coin : IComparable
+	{
+		public Money Amount { get; init; } = Money.Zero;
+		public const int OutputVbytes = 31;
+		public const decimal InputVbytes = 68.5m;
+
+		private int HashCode { get; } = new Guid().GetHashCode();
```

Comment:
won't this be the same for all instances?

Reply:
GUID is random and unique. 

https://docs.microsoft.com/en-us/dotnet/api/system.guid?view=net-5.0

Reply:
Removed the whole class.

Reply:
Removed.